app/src/main/java/org/feature/fox/coffee_counter/data/local/database/dao/UserDao.kt

```diff
@@ -41,6 +50,9 @@ interface UserDao {
     @Query("SELECT * FROM purchase WHERE userId=:id")
     suspend fun getPurchaseListOfUser(id: String): List<Purchase>
 
+    @Query("SELECT * FROM image WHERE userId=:id")
+    suspend fun getImageById(id: String): Image?
+
```

Comment:
Pls add Db at the end of the method name to stay consistent and provide better Overview